### ISSUE TITLE
promote 'metadata' attribute in graph.js and graph.py? to root class

### ISSUE THREAD

**Issue by apeters** [archesproject/arches#586]

**Comment by apeters** [comment]
related to commits 3d8cbea, e2a520f, 541662c, 4553367, d22f646, e31b18c, 2a6ff8b, 32f09a8

**Comment by robgaston** [comment]
I'm getting the following error when trying to append a branch to a graph:

```
Uncaught TypeError: Unable to process binding "text: function (){return graph.root.ontologyclass }"
Message: Cannot read property 'ontologyclass' of undefined
```

I suspect that this new bug may be related; what do you think, @apeters ?

Here's a little screencast of what I'm seeing:

![ug0z1foq5j](https://cloud.githubusercontent.com/assets/1187406/16741890/2bcc34a4-475a-11e6-8b39-f4cb9e3ae2a1.gif)

**Comment by robgaston** [comment]
the bug referenced above is fixed in #590
